snapshots.env

```diff
@@ -1,4 +1,4 @@
-SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_20=https://storage.googleapis.com/circleci-docker-artifacts/data-elixir-omg-tester-plasma-deployer-dev-ba4d320-MIN_EXIT_PERIOD-20-PLASMA_CONTRACTS_SHA-33bb21c16dfa5e33926f5aa7f7b7fb66d3633c1f.tar.gz
-SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_120=https://storage.googleapis.com/circleci-docker-artifacts/data-elixir-omg-tester-plasma-deployer-dev-ba4d320-MIN_EXIT_PERIOD-120-PLASMA_CONTRACTS_SHA-33bb21c16dfa5e33926f5aa7f7b7fb66d3633c1f.tar.gz
-SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_240=https://storage.googleapis.com/circleci-docker-artifacts/data-elixir-omg-tester-plasma-deployer-dev-ba4d320-MIN_EXIT_PERIOD-240-PLASMA_CONTRACTS_SHA-33bb21c16dfa5e33926f5aa7f7b7fb66d3633c1f.tar.gz
-CONTRACT_SHA=33bb21c16dfa5e33926f5aa7f7b7fb66d3633c1f
+SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_20=https://storage.googleapis.com/circleci-docker-artifacts/data-elixir-omg-tester-plasma-deployer-dev-e5c04b9-MIN_EXIT_PERIOD-20-PLASMA_CONTRACTS_SHA-3646712275164482eea35b2b623df30d272de12c-VAULT-false.tar.gz
```

Comment:
you can use the same as in https://github.com/omgnetwork/omg-childchain-v2/blob/master/snapshots.env